### PR TITLE
MdePkg MockUefiLib: Add new mock functions.

### PR DESCRIPTION
**Pull request by v-sbolisetti**
## Description

Added new mock function in MockUefiLib under MdePkg

For details on how to complete these options and their meaning refer to [CONTRIBUTING.md](https://github.com/microsoft/mu/blob/HEAD/CONTRIBUTING.md).

- [ ] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

Unit tests component can call these mock functions success

## Integration Instructions

N/A

### CODE REVIEW

**Comment by v-sbolisetti** [comment]
@VivianNK  @apop5  Please help to review.

**Comment by VivianNK** [comment]
> @VivianNK @apop5 Please help to review.

Hi @v-sbolisetti can you please make this PR to edk2? https://github.com/tianocore/edk2

**Comment by v-sbolisetti** [comment]
@VivianNK  Raised PR under Edk2 :

https://github.com/tianocore/edk2/pull/6344

**Review by apop5 — APPROVED** [review]

**Review by VivianNK — APPROVED** [review]

File: MdePkg/Test/Mock/Include/GoogleTest/Library/MockUefiLib.h [hunk 1]
```diff
@@ -1,7 +1,6 @@
-/** @file
+/** @file MockUefiLib.h
   Google Test mocks for UefiLib
 
-  Copyright (c) 2022, Intel Corporation. All rights reserved.
```
Comment (VivianNK) [inline MdePkg/Test/Mock/Include/GoogleTest/Library/MockUefiLib.h:4]: Please keep the intel copyright

File: MdePkg/Test/Mock/Library/GoogleTest/MockUefiLib/MockUefiLib.inf [hunk 2]
```diff
@@ -1,7 +1,7 @@
-## @file
+## @file MockUefiLib.inf
 # Google Test mocks for UefiLib
 #
-# Copyright (c) 2022, Intel Corporation. All rights reserved.
```
Comment (VivianNK) [inline MdePkg/Test/Mock/Library/GoogleTest/MockUefiLib/MockUefiLib.inf:4]: Please keep the intel copyright.

Comment (v-sbolisetti) [reply MdePkg/Test/Mock/Library/GoogleTest/MockUefiLib/MockUefiLib.inf:4]: Updated the copyright .

File: MdePkg/Test/Mock/Include/GoogleTest/Library/MockUefiLib.h (hunk 1 above)
Comment (v-sbolisetti) [reply MdePkg/Test/Mock/Include/GoogleTest/Library/MockUefiLib.h:4]: Updated the copyright .

**Pull request merged**